app/src/main/kotlin/dev/msfjarvis/aps/ui/firstrun/fragments/CloneFragment.kt

```diff
@@ -1,32 +1,47 @@
 /*
  * Copyright © 2019-2020 The Android Password Store Authors. All Rights Reserved.
- * SPDX-License-Identifier: GPL-3.0-only
+ *  SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Seeing changes like this across a bunch of files, please revert them.

Reply:
Sure